<launch>
  <node cwd="node" name="rosbag_node" pkg="rosbag" type="record"
	output="screen" args="-o $(find aauship_control) /imu /gps1 /kf_statesnew /lli_input /kf_attitude"/>

  <node cwd="node" name="lqr_node" pkg="aauship_control" type="lqr_node" output="screen"/>
</launch>